### ISSUE TITLE
Implement first iteration of Dialog (Modal)

### ISSUE THREAD

**Issue by trybern** [FHIDev/Fhi.Designsystem#218]
### Acceptance criteria

- It should be implemented in accordance [with the designs in Figma](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=9491-5456&p=f&t=iCpOVQHgxQgyMuwV-11)
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs (e.g form interactions)
- It should have appropriate component level design tokens that references semantic, and if appropriate, primitive level tokens.

<img width="1344" height="1254" alt="Image" src="https://github.com/user-attachments/assets/15aa4ff3-215f-4f7a-8c38-0c77e1074531" />

### Additional information

- [Dialog (Modal) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/), fra ARIA Authoring Practices Guide (APG)
